projects/addon-mobile/components/mobile-calendar/mobile-calendar.style.less

```diff
@@ -85,6 +85,34 @@
         letter-spacing: -0.025625rem;
         text-align: center;
     }
+
+    /**
+      * @note: don't use transparent
+      * https://www.w3.org/TR/css-images-4/#ref-for-color-stop-25
+      * No browser renders transparent as rgba(255, 255, 255, 0),
+      * that's completely wrong, transparent is always rgba(0,0,0,0),
+      * as defined in the CSS Color 3 specification.
+      * However, a few years ago W3C changed how color interpolation works in gradients
+      * and specified it should happen in a premultiplied RGBA space, exactly to fix
+      * this issue and make interpolation with transparent work as expected.
+      * Looks like other browsers have implemented this change, but Safari hasn't yet.
+      *
+      * For most browsers ~ transparent === rgba(255,255,255,0)
+      * But Safari renders it as ~ transparent === rgba(0,0,0,0)
+     */
+    &:after {
+        /* Safari 9+ */
+        @supports (-webkit-hyphens: none) {
+            & {
+                background-image: linear-gradient(
+                    to right,
+                    var(--tui-base-01) 0%,
```

Comment:
But this effectively disables the gradient. What we need is `rgba(255, 255, 255, 0)` here, and in dark theme it has to be `rgba(0, 0, 0, 0)`. And it probably should be fixed at the mixin, even though it's the only place we use it. Every browser understands gradient from transparent to any color, it's just Safari that renders transparent as `rgba(0, 0, 0, 0)` no matter what it stands against. So whether this is due to changes to interpolation or not, the comment is just confusing, the actual issue is effectively what I wrote in the previous sentence.